sysutils/jello/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    jello
+version                 1.5.4
+revision                0
+
+homepage                https://kellyjonbrazil.github.io/jello/
+
+description             Filter JSON and JSON Lines data with Python syntax
+
+long_description        Jello is similar to jq in that it processes JSON and \
+                        JSON Lines data, except Jello uses standard Python \
+                        dict and list syntax.
+
+categories              sysutils textproc python
+license                 MIT
+maintainers             {ajhall.us:macports @ajhall} \
+                        openmaintainer
+supported_archs         noarch
+
+checksums               rmd160  265e60aabc0d550e4cd30405ba0d145277b42aef \
+                        sha256  6e536485ffd7a30e4d187ca1e2719452e833f1939c3b34330d75a831dabfcda9 \
+                        size    25574
+
+python.default_version  310
+
```

Comment:
this is missing a build dependency on `setuptools` (see [here](https://github.com/kellyjonbrazil/jello/blob/7dbac53cfd3a1ff5b130c6cbc033609e3fb1f0ad/setup.py#L1)) - not sure why the CI doesn't complain about it...

Reply:
Thanks, fixed.